src/vmware/HISTORY.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 5.0.0 (2022-12)
+
+- Update to AVS 2022-05-01 API
+- Add `az vmware cluster list-zones`
+- Add `--affinity-strength` and `--azure-hybrid-benefit-type` parameters to `az vmware placement-policy vm-host create`
+- Add `--affinity-strength` and `--azure-hybrid-benefit-type` parameters to `az vmware placement-policy vm-host update`
+- Add `--express-route-id` parameter to `az vmware authorization create`
+- Add `vmware addon arc` command group
+- Add `az vmware location check-trial-availability`
+- Add `az vmware location check-quota-availability`
```

Comment:
Could you please add some description for these new commands and new parameters?

Reply:
There's a separate help.py file that contains help for each of these commands and parameters. I'd also like to keep the HISTORY.md file minimal for readability.